tests/modules/blockedclient.c

```diff
@@ -188,6 +188,20 @@ int do_rm_call(RedisModuleCtx *ctx, RedisModuleString **argv, int argc){
     return REDISMODULE_OK;
 }
 
+int do_fake_bg_true(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
let's add a short comment describing what it does (reply from blocked client without actually creating a thread)